mm2src/mm2_main/src/rpc/dispatcher/dispatcher.rs

```diff
@@ -214,13 +218,9 @@ async fn dispatcher_v2(request: MmRpcRequest, ctx: MmArc) -> DispatcherResult<Re
         "ibc_chains" => handle_mmrpc(ctx, request, ibc_chains).await,
         "ibc_transfer_channels" => handle_mmrpc(ctx, request, ibc_transfer_channels).await,
         "withdraw_nft" => handle_mmrpc(ctx, request, withdraw_nft).await,
-        "start_eth_fee_estimator" => handle_mmrpc(ctx, request, start_eth_fee_estimator).await,
-        "stop_eth_fee_estimator" => handle_mmrpc(ctx, request, stop_eth_fee_estimator).await,
-        "get_eth_estimated_fee_per_gas" => handle_mmrpc(ctx, request, get_eth_estimated_fee_per_gas).await,
```

Comment:
If get_eth_estimated_fee_per_gas rpc is removed what means we could get such estimations only by subscription to event streaming.
I thought subscription to event streaming is an alternative option and there is always a direct method.

Reply:
yup this should be the case. streaming should be another option indeed.
i removed them since the start/stop pattern really imitates the streaming behaviour (but you need to send a request to get the gas est instead of it being streamed). and we also keep an extra state in the coin that's kinda similar to the eth_est_streamer state so feels like logic is done twice here.

im thinking of reintroducing it but have it as a direct call to `get_eip1559_gas_fee` right away. with no caching mechanism, so also no start/stop endpoints.

Reply:
rpc re-introduced here: [d387d1b](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2172/commits/d387d1bfc11a1f0d150402f6c66a0210a7105b44) without caching from mm2 side though. Is this good enough?

i could imagine it being useful for mm2s intended to be ran only as a cli (with with no client on top).

Reply:
Yes I think a non-caching rpc is useful for those who cannot use streaming for some reasons (bots, simple clients)

Reply:
Also, as now no more caching of gas api results someone may start making frequent calls and overload the backend gas fee provider.
But I guess to access backend providers we will use the komodo defi proxy where this could be rate limited, so there is no such issue, right?@onur-ozkan

Reply:
yup, we seem to have once in place
https://github.com/KomodoPlatform/komodo-defi-proxy/blob/e382d2945887fa2b61db39c3daf13e8daf135560/src/proxy/http/mod.rs#L46-L50